generic/stress-ng.py

```diff
@@ -154,7 +154,7 @@ def test(self):
         collect_dmesg(self)
         ERROR = []
         pattern = ['WARNING: CPU:', 'Oops', 'Segfault', 'soft lockup',
-                   'Unable to handle', 'Hard LOCKUP']
+                   'Unable to handle', 'ard LOCKUP']
```

Comment:
@harish-24 wouldn't comparing with ignore case be better option ?

Reply:
@narasimhan-v Yes, will adapt that.

Reply:
@narasimhan-v On a second thought, all other strings are case sensitive and messages like `WARNING: CPU`, `Segfault` is better to check as is than to change case and check. So on a wider thought I think existing fix would be better.

Reply:
ok